client/blocks/follow-button/test/button.js

```diff
@@ -1,10 +1,13 @@
-import { expect } from 'chai';
-import { render } from 'enzyme';
+/**
+ * @jest-environment jsdom
+ */
+
+import { render, screen } from '@testing-library/react';
 import FollowButton from '../button';
 
 describe( 'FollowButton', () => {
 	test( 'should apply a custom follow label', () => {
-		const wrapper = render( <FollowButton followLabel="Follow Tag" /> );
-		expect( wrapper.text() ).to.contain( 'Follow Tag' );
+		render( <FollowButton followLabel="Follow Tag" /> );
+		expect( screen.getByText( 'Follow Tag' ) );
```

Comment:
Looking up by test makes sense 👍  However, could you add some sort of assertion here to make the test explicit? Right now, we implicitly check for the existence of an element with such text. Also, as per `jest`'s [docs](https://jestjs.io/docs/expect#expectvalue):

> You will rarely call expect by itself. Instead, you will use expect along with a "matcher" function to assert something about a value.

There are multiple ways we could assert the existence of the element here, for example:

* Assert that it's defined - `.toBeDefined()`
* Assert that it exists in the document tree: `.toBeInTheDocument()`
* Assert that it's visible - `.toBeVisible()`

I think I lean toward the third one because it's the most specific (theoretically an element could be rendered but be invisible), but let me know what you think about this.